lib/HttpClient.ts

```diff
@@ -124,8 +129,22 @@ export class HttpClient {
         return this.request('OPTIONS', requestUrl, null, additionalHeaders || {});
     }
 
-    public get(requestUrl: string, additionalHeaders?: ifm.IHeaders): Promise<HttpClientResponse> {
-        return this.request('GET', requestUrl, null, additionalHeaders || {});
+    public async get(requestUrl: string, additionalHeaders?: ifm.IHeaders, allowRedirects: boolean = true, maxRedirects: number = 50): Promise<HttpClientResponse> {
+        let response = await this.request('GET', requestUrl, null, additionalHeaders || {});
+
+        while (response.message.statusCode === HttpCodes.ResourceMoved
```

Comment:
Shouldn't we handle all of these? 

    MovedPermanently = 301,
    ResourceMoved = 302,
    TemporaryRedirect = 307,
    PermanentRedirect = 308,

Reply:
I wasn't sure about this but it makes sense to handle them all. I added this change.